include/rotatingMHD/convergence_test.h

```diff
@@ -214,24 +214,53 @@ class ConvergenceTestData
    */
   bool save(const std::string &file_name);
 
-
-
 private:
 
+  /*!
+   * @brief Method which formats the columns of the convergence table.
+   *
+   * @details The column are printed in scientific notation with a precision of two. The convergence
+   * rates are also evaluated.
+   *
+   */
   void format_columns();
 
   const ConvergenceTestType type;
 
+  /*!
+   * @brief Number of cycles performed in the convergence test. This number is equal to
+   * the number of lines in the convergence table.
+   */
   unsigned int level;
```

Comment:
I would revert this back to match the global refinement levels, as it from a quick visual inspection one knows if the table corresponds to a spatial or a temporal convergence test since the time step column is right next to it. Furthermore, one can more easily modify a `prm` file to replicate old convergence results just by a visual inspection of said table. Another member called `cycle` can be added to do what `level` is doing in this branch. Although I find `cycle` (how `level` is currently being used) somewhat redundant.

Reply:
Renamed this variable to `refinement_level` in 3536dc4.